.github/workflows/preview.yml

```
@@ -0,0 +1,31 @@
+name: Deploy PR previews
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+
+concurrency: preview-${{ github.ref }}
```

The syntax for using concurrency might have since changed from the example given in `rossjrw/pr-preview-action`
Reading the [github actions docs](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#concurrency), I can only see examples that look like this:
```suggestion
concurrency: 
  group: preview-${{ github.ref }}
  cancel-in-progress: true
```